### DISCUSSION TITLE
The CommunityToolkit team is doing a fantastic job

### DISCUSSION THREAD

**Discussion by davefxy** [CommunityToolkit/Maui#2014] [General]
I have struggled in the past with trying to implement UI elements that didnt exist. This team has eliminated these creative hurtles. For me CommunityToolkit has made creating .Net Maui apps so much easier. Since Net 8 I have found creating a mobile app and making a UI with the help of CommunityToolkit so refreshing. Thank you so much.

**Comment by bijington** [comment]
Thank you very much for the kind words

**Comment by CliffAgius** [comment]
Thank you for the support I am closing this discussion just to tidy up the list.
